adapter-excel/src/test/java/ch/prevo/open/node/adapter/excel/MatchForCommencementNotificationWriterTest.java

```diff
@@ -59,15 +60,18 @@ public void terminationNotificationShouldGenerateExcelFile() throws IOException,
         employmentCommencement.setCapitalTransferInfo(transferInformation);
         employmentCommencement.setEmploymentInfo(employmentInfo);
 
-        final FullTerminationNotification notification = new FullTerminationNotification();
+        final FullMatchForCommencementNotification notification = new FullMatchForCommencementNotification();
         notification.setPreviousRetirementFundUid("CHE-109.740.084-Baloise-Sammelstiftung");
         notification.setTerminationDate(LocalDate.of(2018, 6, 30));
         notification.setEmploymentCommencement(employmentCommencement);
 
-        final String filename = File.createTempFile("openprevo_text", ".xlsx").getAbsolutePath();
+        final String filenamePrefix = File.createTempFile("openprevo_text", "").getAbsolutePath();
+        System.setProperty(FILE_PROPERTY, filenamePrefix);
+        final String filename = String.format("%1$s_%2$tY-%2$tm-%2$td.xlsx", filenamePrefix, LocalDate.now());
+        new File(filename).deleteOnExit();
 
         // when
-        final TerminationNotificationWriter writer = new TerminationNotificationWriter(filename);
+        final MatchForCommencementNotificationWriter writer = new MatchForCommencementNotificationWriter();
         writer.append(notification);
         writer.close();
 
```

Comment:
These three lines could be replaced with `new ExcelMatchNotificationListener().handleMatchForCommencementNotification(notification)`: this would test the current usage, and add coverage to ExcelMatchNotificationListener (currently 0). Might be seen as "unclean" though, as it involves two classes...